aspnetcore/fundamentals/static-files.md

```diff
@@ -66,6 +66,8 @@ The parameterless `UseStaticFiles` method overload marks the files in web root a
 
 [!code-cshtml[](static-files/samples/1x/Views/Home/Index.cshtml?name=snippet_static_file_wwwroot)]
 
+In the preceding code, the tilde character (`~`) represents the ASP.NET Core web root operator. For more information, see [Web root](xref:fundamentals/index#web-root). 
```

Comment:
Again, don't invent the term "web root operator" and the `~` on its own doesn't mean anything. A "virtual path" begins with `"~/"`.